src/board.py

```diff
@@ -0,0 +1,35 @@
+class Board:
+    def __init__(self):
+        self.starter_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+
+    def count_marks_in_board(self, board, player_one, player_two):
+        for mark in board:
+            total_marks_on_board = board.count(player_one) + board.count(player_two)
+        return total_marks_on_board
+
+    def get_current_player(self, total_marks_on_board):
+        if total_marks_on_board == 0:
+            return "X"
+        elif total_marks_on_board % 2 == 0:
+            return "X"
```

Comment:
Can you abstract away these string literals? Also, it's a bit of a smell to return the same value from two additional branches. Are you sure you need that first condition? What does `0 % 2` evaluate to?

Reply:
In this last commit ([235bb0a](https://github.com/t-keazirian/tic-tac-toe/pull/12/commits/235bb0a5c0d638354723897e92e409ecd64a4200)), I moved `player_one` and `player_two` variables from the Game class to the Board class, which I believe makes more sense and now they are not string literals.